wifimgr.py

```diff
@@ -15,6 +15,17 @@
 server_socket = None
 
 
+def _unquote(s):
+    r = s.replace('+', ' ').split('%')
+    for i in range(1, len(r)):
+        s = r[i]
+        try:
+            r[i] = chr(int(s[:2], 16)) + s[2:]
+        except Exception:
```

Comment:
please do not add new code with `except:` or `except Exception:`, rather catch the more specific exceptions.